src/ui/components/common/content_actions.rs

```diff
@@ -63,8 +60,11 @@ where
   });
   let crosspost_label = "Crosspost";
 
+  // TODO a very strange bug, where when using Show, you have to use a stored value
+  let creator_stored = StoredValue::new(creator.clone());
```

Comment:
Really weird one, and I wasted too much time on it. If the `<ReportButton` is outside of `<Show`, this isn't necessary. Inside show, it needs this specific weird cloning.

Reply:
[This part of the Leptos book should make the issue a bit clearer.](https://book.leptos.dev/interlude_projecting_children.html)

Reply:
Nice, thx.